<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>offscreen</class>
 <widget class="QWidget" name="offscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: black;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: white;</string>
   </property>
   <property name="text">
    <string>Off Screen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #444444;
color: white;
font: bold 16pt &quot;Arial&quot;;
border-radius: 10px;
padding: 6px;
</string>
   </property>
   <property name="text">
    <string>PowerButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #444444;
color: white;
font: bold 16pt &quot;Arial&quot;;
border-radius: 10px;
padding: 6px;
</string>
   </property>
   <property name="text">
    <string>Bypass Wait</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectChargerButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #444444;
color: white;
font: bold 16pt &quot;Arial&quot;;
border-radius: 10px;
padding: 6px;
</string>
   </property>
   <property name="text">
    <string>Connect Charger?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: white;</string>
   </property>
   <property name="text">
    <string>(Hold 5 Seconds)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: white;</string>
   </property>
   <property name="text">
    <string>Connects / Disconnects Charger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: white;</string>
   </property>
   <property name="text">
    <string>Starts Device Instantly</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: white;</string>
   </property>
   <property name="text">
    <string>Note: Don't Forget to Create and Set a Profile as Active!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: white;</string>
   </property>
   <property name="text">
    <string>xx</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
